src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import java.util.ArrayList;
+import duke.tasks.*;
```

Comment:
>  Imported classes should always be listed explicitly.

Would you agree that you should change to import the individual tasks instead of using a wild card import